notebooks/Mitigating Disparities in Ranking from Binary Data.ipynb

```diff
@@ -77,9 +77,140 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
I don't think you meant to check in the changes here. Please double check this one and all other notebook changes.

Reply:
Ouch. No I didn't. I hadn't spotted the number of changed files exploding :-(